File: atcorder/abc149/c/main.py
import sys
input = sys.stdin.readline
sys.setrecursionlimit(10**7)
inf = 10**17
mod = 10**9+7

def is_prime(n):
    for i in range(2, n + 1):
        if i * i > n: break
        if n % i == 0: return False
    return n != 1


x = int(input())

while 1:
    if is_prime(x):
        print(x)
        exit()
    x += 1

File: atcorder/abc149/c/main.cpp
#include <bits/stdc++.h>
using namespace std;

#define ALL(v) v.begin(), v.end()
#define V vector
#define P pair
#define ld long double
#define ll long long
#define mod 1000000007
#define IINF INT_MAX
#define INF 1LL << 30

bool is_prime(ll N){
    for(ll i = 2; i * i <= N; i++){
        if(N % i == 0) return false;
    }
    return true;
}


int main() {
    ll x; cin >> x;
    while(1){
        if(is_prime(x)){
            cout << x << endl;
            return 0;
        }
        x++;
    }

    return 0;
}
